operator/deploy/olm-catalog/openshift-ansible-service-broker-manifests/4.3/openshiftansibleservicebroker.v4.3.0.clusterserviceversion.yaml

```diff
@@ -74,6 +74,7 @@ spec:
           - secrets
           - services
           - services/finalizers
+          - endpoints
```

Comment:
Does this need to be added to other `Role` manifests as well?

Reply:
based on our slack conversation it doesn't need to be added to other Roles.